bridge_ui/src/hooks/useIsWalletReady.ts

```diff
@@ -56,7 +64,18 @@ function useIsWalletReady(
       if (!isEVMChain(chainId)) {
         return;
       }
+      if (
+        connectType === ConnectType.WALLETCONNECT &&
+        EVM_RPC_MAP[correctEvmNetwork] === undefined
+      ) {
+        // WalletConnect requires a rpc url for this chain
+        // Force user to switch connect type
+        disconnect();
+        return;
+      }
+
       try {
+        console.log("switching");
```

Comment:
Not sure where we are on log messages.  Doesn't bother me.  But a one word log may not provide much context.

Reply:
agree, we should probably remove the log